Recolor.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class to recolour a transparent image preserving transparency
+ *
+ * @author Robin Andrews (prandrews@hotmail.co.uk)
+ * @author David Yell (neon1024@gmail.com)
+ * @license MIT
+ */
+class Recolor {
```

Comment:
This is the name of the class. We could also extend from a parent class if we were working with something else we wanted to enhance.